.gitignore

```diff
@@ -31,3 +31,11 @@ package-lock.json
 .nvmrc
 .rspec
 .tool-versions
+
```

Comment:
We probably can remove the folders `public/packs`, `public/packs-test` and `public/webpack` as well from this file, right?

Reply:
removed the ones I found